linux_os/guide/system/permissions/files/file_permissions_etc_audit_auditd/rule.yml

```diff
@@ -0,0 +1,37 @@
+documentation_complete: true
+
+prodtype: fedora,rhel8
+
+title: 'Verify Permissions on /etc/audit/auditd.conf'
+
+description: |-
+    {{{ describe_file_permissions(file="/etc/audit/auditd.conf", perms="0640") }}}
+
+
+rationale: |-
+    Without the capability to restrict the roles and individuals that can select which events
+    are audited, unauthorized personnel may be able to prevent the auditing of critical
+    events. Misconfigured audits may degrade the system's performance by overwhelming
+    the audit log. Misconfigured audits may also make it more difficult to establish,
+    correlate, and investigate the events relating to an incident or identify
+    those responsible for one.
+
+severity: medium
+
+identifiers:
+    cce@rhel8: CCE-85871-2
+
+references:
+    nist: AU-12b
+    disa: CCI-000171
+    srg: SRG-OS-000063-GPOS-00032
+    stigid@rhel8: RHEL-08-030610
+
+template:
+    name: file_permissions
+    vars:
+      filepath: ^/etc/audit/rules.d/auditd.conf$
```

Comment:
the dot in a regex would match any character, if you want to match a real dot, you need to escape it by a backslash, and also quote the string